Add more tips on resume writing

rudi-c opened the pull request.

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -78,24 +64,50 @@ Volunteer experience
-- if you have more than 4 bullet points for one section, it'll become a big block of text and will become harder to scan as well
+As much as possible, don't list your skills without context proving you have them. Anyone can write "good communicator" on their resume, and unless you talk about something you did that needed very good communication skills, that line is essentially useless. In general, just don't list "good communicator" as a skill and instead talk about your communication skills when you talk about your experie ...
 
+Similarily, show that you know programming languages by mentioning them in the work you've done with them and don't just list a bunch of them at the top of your resume. It's useful to have them all in one place of course. But as a general rule of thumb, any language/technology/acronym that you put in a list should also appear elsewhere on your resume.
```
I don't actually have the list anymore and it's been working okay for me

"It's useful" -> "It can be useful"

Also, sometimes you can have things in the list that you know but haven't used in recent interesting projects that made it onto your resume. In those cases I think it's fine to not have it appear somewhere else, but also know that it isn't making much of an impact (oh you talked about that a bit in the paragraph below, but that's a much more nerdy example - Haskell for fun challenge vs just having some smaller projects that don't matter as much anymore. It might be more inclusive to generalize that example a bit)

rudi-c replied:
True, it's generally useful to show any signs of playing around and learning new tools. I'll put a couple different vastly examples (off the top of my head Vue.js, Perl, Haskell, Google Maps API, TensorFlow, Processing - is that too much?).

For the language list, I'm worried about the recruiters who are trained to do initial scans of "knows at least 2 of the following mainstream languages". Unfortunately, these recruiters are often found at places that people are trying to get into (because those places are popular, therefore attracts lots of people who don't even know programming!).

evykassirer replied:
I think in this file just be vague about "smaller or less relevant projects" - I find both resume things and side project things separately overwhelming and there's no need to put them both in this file

it's interesting how in editing your PRs I'm realizing how sensitive I've gotten to advice on job hunting - I seek it all out and try to absorb it all but I just get overwhelmed by it pretty easily too. I'm definitely trying to actively combat that here but it'd be interesting to see how someone similar to me but not me would feel reading my own advice, and how overwhelming it would all be and what I could do to make it better

rudi-c replied:
Yeah, I'm interested to. I really want to give useful advice, but I feel like I have no data on the impact of my writing when it comes to something like "does this end up scaring people away?". Wondering how we can get good feedback, there's the problem of getting the right people to speak up.

At some point we should post this on reddit for feedback. People tend to express their disagreement is less-than-productive ways so it's kinda scary lol. But I feel a lot of the cynicism is a response to stress and feeling that expectations placed on them are too high. Which is very similar to the feeling of being overwhelmed.

evykassirer replied:
reddit sounds good to me - we can also ask people in fb groups too (I don't even mind double posting in some groups if we've added a bunch more content)

(if you can let me post when I feel ready I'd appreciate)

rudi-c replied:
Of course!

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -78,24 +64,50 @@ Volunteer experience
+
+In general, it might seem like writing more will make you seem more impressive, but if you write tons and tons only to fill space, it won't be impressive because it will be obvious that it's just filler. Short resumes clearly and concisely demonstrate your qualifications and are easier for a recruiter or potential employer to scan and understand. You do not want to dilute your impressive accompli ...
 
+How do you know if you're being too verbose? If you do have a two-page resume, you should definitely at least do the exercise of shortening it until it fits into one page. Just think of it as a way to practice being concise, and ask your friends to cross out what they think is the least interesting if needed. You might not like the end result and decide not to keep it. That's fine. But you might  ...
```
ha! that's actually how I ended up with my one page

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -78,24 +64,50 @@ Volunteer experience
 
+How do you know if you're being too verbose? If you do have a two-page resume, you should definitely at least do the exercise of shortening it until it fits into one page. Just think of it as a way to practice being concise, and ask your friends to cross out what they think is the least interesting if needed. You might not like the end result and decide not to keep it. That's fine. But you might  ...
+
+It's useful to pay attention to the _information density_ of each of your sentences. How much information do I provide the reader? You can do this analysis on a word-by-word level. Here are a few examples inspired by resumes I've seen in real life.
```
this section is super cool

rudi-c replied:
Thanks! Should it stay in this page or go in a separate "how to condense your resume" page?

evykassirer replied:
I'm a fan of separate pages esp since this file is pretty big - makes things more easy to digest I think? Like how small PRs are easier to review ^_^ (thanks!)

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -78,24 +64,50 @@ Volunteer experience
+- _Communicated ideas and concepts clearly while reviewing the main principles to build a solid foundation in web development_. This sentence provides no information. In the first half, it's expected that a teacher would communicate clearly, it's not worth mentioning unless you have concrete evidence that demonstrates the teaching quality. The second half is already part of teaching a class.
+- _Delivered lessons on using HTML, CSS and general web design principles_. This provides some information about what was taught concretely, though largely guessable.
+
+The above could be condensed into "Designed and taught an introductory class on web design using HTML and CSS. Learned how to communicate concepts clearly to students with no prior experience".
```
I think it's worth noting that if you have very little experience, writing out some of these things a little more verbosely is okay. You do want to end up with a full page (though that page can have lots of whitespace). But being aware of which of these details the reader actually cares about is still helpful

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -78,24 +64,50 @@ Volunteer experience
+The above could be condensed into "Designed and taught an introductory class on web design using HTML and CSS. Learned how to communicate concepts clearly to students with no prior experience".
+
+
+### How to talk about things
```
this reordering of sections makes a lot of sense, thanks :)

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -104,17 +116,16 @@ Consistency:
 - make sure your dates are in the same format (e.g. Aug 2017 or September '17)
 - in general, the way you format each section and the way you talk about things should be consistent - this makes it easy to find information and to scan your resume
-- in case you're thinking this right now - yes I could have better consistency in this guide because I'm not alawys doing the things I literally just mentioned ...but it's a lot harder for a thing this big and I guess I don't care enough ^_^
+- in case you're thinking this right now - yes I could have better consistency in this guide because I'm not alawys doing the things I literally just mentioned ...but it's a lot harder for a thing this big and I guess I don't care enough ^.^
```
haha you changed my emoji! (that's fine)

rudi-c replied:
The syntax highlighter for Markdown in vim was getting confused by the _. It thought that the entire rest of the file was supposed to be highlighted... ^^'

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -131,3 +142,21 @@ More resources about resumes:
+
+#### Make sure you can print your resume:
+
+Margins should be wide enough that things aren't cut off.
```
this could be a bullet list

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -131,3 +142,21 @@ More resources about resumes:
 
```
resources should be a section at the bottom

rudi-c commented:
Changes:
- Reworded bit on small languages/technologies
- Moved the information density into a separate file, added a comment that it's fine to be more verbose if you have less experience
- Moved links to the bottom
- The link to your resume was broken because it had https://

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -78,24 +64,38 @@ Volunteer experience
-- if you have more than 4 bullet points for one section, it'll become a big block of text and will become harder to scan as well
+As much as possible, don't list your skills without context proving you have them. Anyone can write "good communicator" on their resume, and unless you talk about something you did that needed very good communication skills, that line is essentially useless. In general, just don't list "good communicator" as a skill and instead talk about your communication skills when you talk about your experie ...
 
+Similarily, show that you know programming languages by mentioning them in the work you've done with them and don't just list a bunch of them at the top of your resume. It's useful to have them all in one place of course. But ideally, languages/technologies/acronyms that you put in a list should also appear elsewhere on your resume.
```
it's useful -> it can be useful

rudi-c replied:
Fixed

evykassirer commented on docs/1-Applying/resumes.md:
```diff
@@ -74,9 +74,9 @@ I'd recommend checking out other people's resumes too, as some of them have sect
+Similarily, show that you know programming languages by mentioning them in the work you've done with them and don't just list a bunch of them at the top of your resume. It's useful to have them all in one place of course. But ideally, languages/technologies/acronyms that you put in a list should also appear elsewhere on your resume.
 
-(There are some exceptions to this. For example, you might want to include Haskell to show that you like to challenge yourself intellectually, but your largest Haskell project is smaller than all your other projects. It's fine to include Haskell, as long as you are able to explain how you've used Haskell during an interview if asked. In general, you should be able to talk about anything on your r ...
+We say ideally because you might have dabbled in some new technology just to try it (e.g. some Javascript framework, a functional language, some API, etc). That still signals to the employers that you can learn on your own which is great [see side projects](side_projects.md). It's fine to include them, as long as you are able to explain how you've used them during an interview if asked. In genera ...
```
as we discussed, I don't think you have to give examples of technologies here

evykassirer replied:
also this example is confusing to me - if you do a side project, often that'll be on your resume

rudi-c commented:
I'd rather leave the examples here. I think it's actually not obvious by what is meant by a "technology". In common use, I think we tend to think more of physical inventions like planes, self-driving cars or whatnot. But it's not until you've spent a while in tech that you pick up that people also refer to libraries, languages, programming tools as 'technologies'.

evykassirer commented:
could you list "libraries, languages, programming tools" then? 

(if that doesn't appeal to you then I'll just let it go haha)

rudi-c commented:
Sure, that works for me.
